azure-devops/azext_devops/dev/common/services.py

```diff
@@ -219,6 +219,12 @@ def get_work_item_tracking_client(organization=None):
                                  'work_item_tracking_client.WorkItemTrackingClient')
 
 
+def get_extension_client(organization=None):
+    connection = get_vss_connection(organization)
+    return connection.get_client(VSTS_MODULE +
+                                 'extension_management.v4_1.extension_management_client.ExtensionManagementClient')
```

Comment:
Why 4_1?

Reply:
because I would like to port this to 5_0 as soon as that is available, 4_1 is no longer the edge version because 5_0, 5_1 is out